asserts/snapasserts/validation_sets_test.go

```diff
@@ -1142,3 +1144,220 @@ func (s *validationSetsSuite) TestValidationSetKeyComponents(c *C) {
 	}).(*asserts.ValidationSet))
 	c.Assert(valsetKey.Components(), DeepEquals, []string{"a", "b", "c", "13"})
 }
+
+func (s *validationSetsSuite) TestRevisions(c *C) {
```

Comment:
Is the test case checking the case of presence: invalid?

Reply:
Added a case, although it will not be present in the output.